com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/ArccosindeNode.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class ArccosineNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,
+            "Arccosine",
+            "Out = acos(In);",
+            new ParameterDescriptor("In", TYPE.Vector, Usage.In, new float[] {1f, 0f, 0f, 0f}),
```

Comment:
The default value should be { 1f, 1f, 1f, 1f } so that each of the components get the same default.